### DISCUSSION TITLE
Runtime Error: Numpy is Unavailable

### DISCUSSION THREAD

**Discussion by Nextlvlanalytics** [MaartenGr/BERTopic#676] [Q&A] [answered]
Hi All,

I'm incredibly stuck here. I'm trying to add a topic modeling layer to an analysis that I'm working on and would like to use BERTopic as it seems the most straightforward solution to my task. Where I'm stuck is I keep getting a "RunetimeError: Numpy is not available". 

I have no idea how to resolve this. I already had a tough time trouble shooting HDBSCAN not pip installing correctly so I had to find a .whl to meet the package criteria -- I'm not sure if thats maybe whats causing the error? 

Thanks in advance

**Comment by MaartenGr** [comment, marked as answer]
It might be indeed due to the manual install of HDBSCAN. Previous versions of HDBSCAN had issues with bumpy but that should be resolved with the newest version. I would advise starting from a completely fresh environment and using `pip install --upgrade bertopic` there.

These type of issues are often resolved using fresh environments where no other packages are installed such that the install can start without any major dependencies already installed.
